accepted/00076-configuration-channels-visibility.md

```diff
@@ -0,0 +1,201 @@
+
+After:
+![Configuration Channels Detail Systems AFTER](images/configuration-channels-visibility/Configuration_Channels_Details_Systems_AFTER.png)
+
+
+### `Configuration > Channels > Details > System Groups`
+This page reproduces the same behavior of the `Configuration > Channels > Details > Systems` tab, but for System Groups.
+
+This **new** page is pointed by:
+- the `Channel information` box link of the system groups counting in the `Configuration > Channels > Details > Overview` page: the link will land on the new page at the `Subscribed System Groups` tab
+
+### `Configuration > Channels > System Groups`
+This page represents a complete list of pairs value `Configuration State Channel - System Group`: every value is a link to the specific entity detail (channel or system group).
+
+This **new** page is pointed by:
+- the `Configuration summary` box link of the system groups counting in the `Configuration > Overview` page: the link will land on the new page where all the pairs `Configuration State Channel - System Group` are listed
+- the `Configuration > Channels` list, for every channel entry, from the `Subscribed System Groups` counting link value
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+## Why should we **not** do this?
+
+No real reason for not enhancing the feature adding the missing visibility.
+
+The concrete drawback is: the solution is adding information to a tech-debt affected architecture, spreading the same information in many places (see how many pages/screenshots). Maintaining should not be complicated, but future refactoring of the feature workflow could be more difficult.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## What other designs/options have been considered?
+
+Without investigating deeply, the alternative would be redesigning the feature completely from both the design and technical perspectives: the technical refactoring should get rid of the struts stack and redefine the concept of Configuration Channel and Configuration State Channel used by traditional or salt minion clients. By addressing the technical part, also the workflow could be re-drawn, dropping some pages and instead of adding separated tabs of tables it would be preferrable to have unified modern lists of filterable entries. The drawback of this alternative is that it would take much longer (from 3 to 5 times roughly) than the proposed official solution.
+
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+## What are the unknowns?
+1. There is a real need for representing `registration-states` (as mentioned before) or they can still stay behind the scene?
+2. Salt States coming from Formulas are not yet considered as part of this RFC
+3. This RFC is considering presenting all the sources of the Salt States for a given system. The RFC is **not** considering instead to implement a reverse-engineering to pick every Salt State from a given HighState and build the history of the specific source where it came from.
```

Comment:
Agreed.

I would also add: this RFC is no about tracing each line in the highstate to one of the states assigned directly/indirectly to a system.